core/cmd/remote_client.go

```diff
@@ -412,18 +412,37 @@ func (cli *Client) GetConfiguration(c *clipkg.Context) (err error) {
 	return err
 }
 
-func (cli *Client) ConfigDump(c *clipkg.Context) (err error) {
+func (cli *Client) FetchV2Config() (string, error) {
```

Comment:
Does this need to be exported? If not, how about aligning them with the same name?
```suggestion
func (cli *Client) configDump() (string, error) {
```

Reply:
It looks like it has to be, in order to be accessed from the test file (different package name).

Reply:
You can add a wrapper to `helpers_test.go` to make it available to `package cmd_test` only.